interactem-cli/interactem_cli.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+import json
+import signal
+from uuid import UUID
+
+import typer
+
+from interactem.core.constants import SUBJECT_PIPELINES_RUN, SUBJECT_PIPELINES_STOP
+from interactem.core.events.pipelines import PipelineRunEvent, PipelineStopEvent
+from interactem.core.nats import nc
+
+app = typer.Typer()
+
+@app.command()
+def run(f: str = typer.Option(..., "--file", "-f", help="Path to pipeline.json")):
```

Comment:
shouldn't this be a `pathlib.Path`, not a `str`? try namig it something more descriptive than `f`

https://typer.tiangolo.com/tutorial/parameter-types/path/